yarn.lock

```diff
@@ -7683,12 +7683,12 @@ braces@^2.3.1, braces@^2.3.2:
     split-string "^3.0.2"
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33040&commitId=5796758c59b3328b2f541586a6af4b93b374f42e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Bottom right edge.chromium.png|1596| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|334| Changed|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>